apps/fz_common/lib/fz_net.ex

```diff
@@ -35,12 +27,33 @@ defmodule FzCommon.FzNet do
     end
   end
 
+  def rand_ip(cidr, type) do
+    parsed = CIDR.parse(cidr)
+    first = FzInteger.from_inet(parsed.first) + 1
```

Comment:
Do we want to allow devices to use X.X.X.1 address (usually reserved for a router) in the address range? I think we should start issuing them from X.X.X.2

Reply:
Good question. The router in our case would be the Gateway, so then the question is, do we want the first Gateway to have `.1`? After our convo this morning I would lean towards randomizing it as well. Predictability / determinism could be handled with tunnel DNS, e.g. `firezone.gateway0`

Reply:
But currently, the default gateway IP is `.1`

Reply:
@jamilbk I'm not sure but why would you randomize the gateway address? All client apps would know it anyways, plus it will be part of network settings that are easy to look up. But for sysadmins, it would be more pain to keep remembering its IP.

Reply:
Yeah we can leave it at `.1`. I was thinking more, what should be the general gateway IP allocation strategy? For the second, third, fourth, Nth gateway. Multi-site can't come soon enough ;-)